google-maps/android/src/main/java/com/capacitorjs/plugins/googlemaps/CapacitorGoogleMap.kt

```diff
@@ -475,6 +476,32 @@ class CapacitorGoogleMap(
         )
     }
 
+    fun getLatLngBounds(): LatLngBounds {
+        return googleMap?.projection?.visibleRegion?.latLngBounds
+            ?: throw BoundsNotFoundError()
+    }
+
+    fun getLatLngBoundsJSObject(bounds: LatLngBounds): JSObject {
+        val data = JSObject()
```

Comment:
This should be enclosed in a "bounds" object, like it is on iOS:

```kotlin
val boundsObj = JSObject()

val southwestJS = JSObject()
val centerJS = JSObject()
val northeastJS = JSObject()

southwestJS.put("latitude", bounds.southwest.latitude)
southwestJS.put("longitude", bounds.southwest.longitude)
centerJS.put("latitude", bounds.center.latitude)
centerJS.put("longitude", bounds.center.longitude)
northeastJS.put("latitude", bounds.northeast.latitude)
northeastJS.put("longitude", bounds.northeast.longitude)

boundsObj.put("southwest", southwestJS)
boundsObj.put("center", centerJS)
boundsObj.put("northeast", northeastJS)

val data = JSObject()
data.put("bounds", boundsObj)

return data
```

Reply:
Made the fix on iOS